src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="navigation_logo">
+          <a
+            class="logo_link"
+            href="#"
+          >
```

Comment:
The href attribute of the a tag should not have spaces. It's better to use a meaningful link instead of '#'. If there is no appropriate page, you can use '#top' or '#logo' to make it more meaningful.